### ISSUE TITLE
make page more attractive

### ISSUE THREAD

**Issue by arifamd00** [utilidev/ud-react-template#17]
add some basic style to the web pages that look more attractive. using HTML and CSS

**Comment by kbrock84** [comment]
I agree this would be nice. I would wait until PR #15 is merged hopefully today.

**Comment by kbrock84** [comment]
PR #15 has been merged. @arifamd00 please let me know if you would like to work on this.
